yglu/main.py

```diff
@@ -30,3 +30,21 @@ def process(input, output, filename=None, errors=[]):
         else:
             first = False
         dump(doc, output, errors)
+
+def process_data(input, filename=None)
```

Comment:
Missing column, no? This is what makes the build fails, I guess.

Reply:
Yes, silly mistake.